see the AppVeyor log, check your code please
```
       "C:\projects\cocos2d-x\build\cocos2d-win32.sln" (Build target) (1) ->
       "C:\projects\cocos2d-x\tests\cpp-tests\proj.win32\cpp-tests.vcxproj" (default target) (2) ->
       (ClCompile target) -> 
         ..\Classes\UITest\CocoStudioGUITest\UIEditBoxTest.cpp(322): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\projects\cocos2d-x\tests\cpp-tests\proj.win32\cpp-tests.vcxproj]
         ..\Classes\UITest\CocoStudioGUITest\UIEditBoxTest.cpp(322): error C2146: syntax error: missing ')' before identifier 'position_y' [C:\projects\cocos2d-x\tests\cpp-tests\proj.win32\cpp-tests.vcxproj]
         ..\Classes\UITest\CocoStudioGUITest\UIEditBoxTest.cpp(322): error C3260: 'identifier': skipping unexpected token(s) before lambda body [C:\projects\cocos2d-x\tests\cpp-tests\proj.win32\cpp-tests.vcxproj]
         ..\Classes\UITest\CocoStudioGUITest\UIEditBoxTest.cpp(324): error C2065: 'position_y': undeclared identifier [C:\projects\cocos2d-x\tests\cpp-tests\proj.win32\cpp-tests.vcxproj]
    2717 Warning(s)
    4 Error(s)
Time Elapsed 00:27:17.37
ScriptHalted
```

--- a/tests/cpp-tests/Classes/UITest/CocoStudioGUITest/UIEditBoxTest.cpp
+++ b/tests/cpp-tests/Classes/UITest/CocoStudioGUITest/UIEditBoxTest.cpp
@@ -319,7 +319,7 @@ bool UIEditBoxTestTextHorizontalAlignment::init() {
   
     const auto createEditBox = [this, editBoxSize, visibleOrigin, visibleSize](const std::string& text,
                                       const TextHAlignment alignment,
-                                      const uint position_y) {
+                                      const int position_y) {
         ui::EditBox* editbox = ui::EditBox::create(editBoxSize + Size(0,40), ui::Scale9Sprite::create("extensions/green_edit.png"));
         editbox->setPosition(Vec2(visibleOrigin.x+visibleSize.width/2-50, position_y));
         editbox->setFontColor(Color3B::RED);

